### ISSUE TITLE
[Models] Model training WorldFloods

### ISSUE THREAD

**Issue by gonzmg88** [spaceml-org/ml4floods#16]
- [ ] Implement training pipeline in pytorch lightning porting code from https://gitlab.com/frontierdevelopmentlab/disaster-prevention/cubesatfloods
- [ ] Make it flexible for trying different loss functions
- [ ] Add metrics to W&B logger
- [ ] Plot some image results at the end of each epoch

Nice to have:
- Weights and Biases integration
- Look into dvc and sacred (open-sourced experiment management systems)
